torch/testing/_internal/common_methods_invocations.py

```diff
@@ -80,6 +80,19 @@ def _getattr_qual(obj, name, default=_NOTHING):
 class OpInfo(object):
     """Operator information and helper functions for acquiring it."""
 
+    class AliasInfo(object):
```

Comment:
For consistency with other classes defined in this file, like SampleInput, let's pull this out of OpInfo.